webdriverio/native-demo-app
# Upgrade to latest versions

> **NOTE:** This PR can only be merged when the Appium Boilerplate has been updated.

This PR will:
- update the app to the latest RN version
- update all dependencies
- Use TypeScript
- Add GitHub Actions for Android and iOS
- Adds Dark Mode to the app, see this [tweet](https://twitter.com/wswebcreation/status/1389294891692331022)
- Deeplinking:
  - **Android:** run `adb shell am start -W -a android.intent.action.VIEW -d "wdio://swipe" com.wdiodemoapp` from the command line, it will open the app in the swipe tab
  - **iOS:**  run `xcrun simctl openurl booted wdio://swipe` from the command line, it will open the app in the swipe tab

- **Biometric support:** This is a simple implementation, just to demo/test the usage, see this [tweet](https://twitter.com/wswebcreation/status/1389608689628291080?s=20)
- **Horizontal swiping:** The swiping screen now also has a horizontal swipe option
- **Drag and drop:** A puzzle has been created to learn how to drag and drop, see this [tweet](https://twitter.com/wswebcreation/status/1391086910769180675?s=20)

![webdriverio](https://user-images.githubusercontent.com/11979740/116923343-ce1b6980-ac56-11eb-88af-973d5b65cb29.gif)
